keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlVerificationRepository.kt

```diff
@@ -20,20 +20,15 @@ import com.netflix.spinnaker.keel.resources.ResourceSpecIdentifier
 import com.netflix.spinnaker.keel.resources.SpecMigrator
 import com.netflix.spinnaker.keel.sql.RetryCategory.WRITE
 import com.netflix.spinnaker.keel.sql.deliveryconfigs.deliveryConfigByName
-import org.jooq.DSLContext
-import org.jooq.Field
-import org.jooq.Record1
-import org.jooq.Record4
-import org.jooq.Select
-import org.jooq.SelectOrderByStep
+import org.jooq.*
```

Comment:
I tried and failed to configure IntelliJ to stop doing the wildcard thing, so I just gave up.

Reply:
![image](https://user-images.githubusercontent.com/1323478/110276267-a0cc7b00-7f87-11eb-81c7-889aaa5c0990.png)

![image](https://user-images.githubusercontent.com/1323478/110276316-b8a3ff00-7f87-11eb-812e-7b68b361b808.png)